src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,20 +25,24 @@
   private String user;
   private String text;
   private long timestamp;
+  private String recipient; // Added recipient
 
   /**
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
-  public Message(String user, String text) {
-    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
+   // Add recipient param to the ctor
+  public Message(String user, String text, String recipient) {
+    this(UUID.randomUUID(), user, text, System.currentTimeMillis(), recipient);
   }
 
-  public Message(UUID id, String user, String text, long timestamp) {
+  // Add recipient param to the ctor
```

Comment:
Add comments via javadoc (basically /** ... */ style) instead of // 

// comments belong inside the method for small blocks, javadocs belong on top of methods to describe what it does